_data/simple-icons.json

```diff
@@ -3548,7 +3548,7 @@
         {
             "title": "Intel",
             "hex": "0071C5",
-            "source": "https://www.intel.com"
+            "source": "https://www.intel.ie/"
```

Comment:
Is it worth keeping this as `intel.com` to keep the 'global' website listed as source?

Reply:
No harm in using a localised site but also no harm in leaving it as-is - in this case, I didn't want to assume the same SVG was available at intel.com. Updating now ...